### ISSUE TITLE
Interactive element does not meet minimum size nor spacing - V2 Carousel

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#9637]
# Motivation

Siteimprove is reporting "Interactive element does not meet minimum size nor spacing" for v2 carousels. This is a AA compliance issue.

# Siteimprove info

**Technical doc**
https://alfa.siteimprove.com/rules/sia-r113

Siteimprove link to existing issue - https://my2.siteimprove.com/Inspector/1009758/A11y/Page?pageId=31228117523&impmd=264ED37B65FA6186129334A508391970&conf=a+aa#/sia-r113/failed/WGNObUNiSFE=;ZGU0MTdkOTA=

Examples
- https://pentacrestmuseums.uiowa.edu/serving-butter-pats
- https://hawkeyemarchingband.uiowa.edu/about/history

**Interactive element does not meet minimum size nor spacing**
Issue
2.5.8: Target Size (Minimum)
Interactive elements, like buttons or links, should be large enough or be positioned, so they can easily be clicked or tapped by users.

Elements should be at least 24 by 24 pixels, or if smaller, be positioned with enough spacing to not intersect with other interactive elements.

**How to fix this issue:**
Enlarge the interactive element to be at least 24 by 24 pixels.
or
Adjust the spacing so a 24-pixel diameter circle centered on the bounding box of an undersized interactive element does not intersect with another interactive element or its circle.
Size of element:
38 x 5 pixels

**Comment by bspeare** [comment]
OK, after looking at this again, the error is coming because the arrows are on top of the dots if the dots fill up the entire width of the image.  The minimum height issue is valid on mobile, so I'm going to make a few adjustments based on this new information. 

<img width="1363" height="844" alt="Image" src="https://github.com/user-attachments/assets/25f1c772-e0cf-4b0c-9cb6-8b4c0fbb3d79" />

<img width="1387" height="852" alt="Image" src="https://github.com/user-attachments/assets/c26212ff-44eb-4d7c-8af7-30f5ef1879ca" />
